### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by arhamkhnz**
Updated README.md to document the `filterCvParams` property in `react-perspective-cropper`.

### CODE REVIEW

**Comment by arhamkhnz** [comment]
Please review my PR I've updated documentation for `filterCvParams` to allow users to crop with custom filtering options. for example in case user wants cropped image without filter

**Comment by giacomocerquone** [comment]
Hey there, I'm on vacation right now but I still wanted to write a couple of words to thank you for the pull request. Since it's just an edit on the readme, it can definitely wait my return 🙂

File: README.md [hunk 1]
```diff
@@ -52,6 +52,29 @@ export interface CropperProps {
...
+    thMeanCorrection: 15,                // Adjusts the mean for thresholding.
+    thBlockSize: 25,                     // Determines the size of the block for adaptive thresholding.
+    thMax: 255,                          // Sets the maximum value for thresholding.
+    grayScale: true                      // Converts the image to grayscale.
+}
+
+
+// Configuration for image processing that can be passed to the function.
+const filterParams = { preview: false, filterCvParams};
+```
```
Comment (giacomocerquone) [inline README.md]: Can you please specify the function that accepts the filterCvParams and the preview values?
That would be the cropperRef.current.done function

Comment (arhamkhnz) [reply README.md]: I've updated the README to specify the cropperRef.current.done function accepts the `filterCvParams`

**Review by giacomocerquone — APPROVED** [review]
thank you

**Pull request merged**